src/commands.rs

```diff
@@ -53,6 +60,43 @@ pub fn handle_run_command(model_dir: &PathBuf) -> Result<()> {
     Ok(())
 }
 
+/// Handle the `example run` command.
+pub fn handle_example_run_command(name: &str) -> Result<()> {
+    // Find the subdirectory in EXAMPLES_DIR whose name matches `name`.
+    let sub_dir = EXAMPLES_DIR.get_dir(name).context("Directory not found.")?;
+
+    // Creates temporary directory
+    let temp_dir = TempDir::new().context("Failed to create temporary directory.")?;
+
+    // Copies the contents of the subdirectory to the temporary directory
+    for entry in sub_dir.entries() {
+        match entry {
+            DirEntry::Dir(d) => {
+                for sub_entry in d.entries() {
+                    match sub_entry {
+                        DirEntry::File(f) => {
+                            let file_name = f.path().file_name().unwrap();
+                            let file_path = temp_dir.path().join(file_name);
+                            fs::write(&file_path, f.contents())?;
+                        }
+                        DirEntry::Dir(_) => {
+                            return Err(anyhow::anyhow!("Nested directories are not supported."));
+                        }
+                    }
+                }
+            }
```

Comment:
As there arern't any subdirectories for the `simple` example and there shouldn't be, let's just disallow this for now:

```suggestion
            DirEntry::Dir(_) => panic!("Subdirectories in examples not supported"),
```

If we decide we need it we can sort the nesting issue at the same time.